hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/TestContainerOperations.java

```diff
@@ -188,19 +188,16 @@ public void testDatanodeUsageInfoCompatibility() throws IOException {
 
   @Test
   public void testDatanodeUsageInfoContainerCount() throws Exception {
-    List<? extends DatanodeDetails> dnList = cluster().getStorageContainerManager()
-            .getScmNodeManager()
-            .getAllNodes();
+    NodeManager nodeManager = cluster().getStorageContainerManager().getScmNodeManager();
+    List<? extends DatanodeDetails> dnList = nodeManager.getAllNodes();
 
     for (DatanodeDetails dn : dnList) {
-      final int expected = cluster().getStorageContainerManager().getScmNodeManager().getContainers(dn).size();
-
       List<HddsProtos.DatanodeUsageInfoProto> usageInfoList =
               storageClient.getDatanodeUsageInfo(
                       dn.getIpAddress(), dn.getUuidString());
 
       assertEquals(1, usageInfoList.size());
-      assertEquals(expected, usageInfoList.get(0).getContainerCount());
+      assertEquals(nodeManager.getContainers(dn).size(), usageInfoList.get(0).getContainerCount());
```

Comment:
This change in `TestContainerOperations` seems completely unrelated to the rest of the PR.